openvdb/math/Coord.h

```diff
@@ -578,8 +591,24 @@ operator<<(std::ostream& os, const CoordBBox& b)
 } // namespace OPENVDB_VERSION_NAME
 } // namespace openvdb
 
+////////////////////////////////////////
+
+// template specialization of std::hash with Coord, which
+// allows for Coord to be used as the key in std::unordered_map
+namespace std {// injected in namespace std
+
+template<>
+struct hash<openvdb::math::Coord>
+{
+    using argument_type = openvdb::math::Coord;
+    using result_type = std::size_t;
+    result_type operator()(const argument_type& ijk) const noexcept { return ijk.hash<>(); }
```

Comment:
ICC complains about this (I'm not sure why):
```
openvdb/math/Coord.h(605): warning #955: ambiguous class member reference --
    function template "openvdb::v6_0::math::Coord::hash" (declared at line 257)
    used in preference to type "std::hash<openvdb::v6_0::math::Coord>::hash" (declared at line 601)

    result_type operator()(const argument_type& ijk) const noexcept { return ijk.hash<>(); }
```
I suggest changing it to
```
template<>
struct hash<openvdb::math::Coord>
{
    using Coord = openvdb::math::Coord;
    using argument_type = Coord;
    using result_type = std::size_t;
    result_type operator()(const Coord& ijk) const noexcept { return ijk.Coord::hash<>(); }
};
```

Reply:
done